Dockerfile

```diff
@@ -13,7 +13,13 @@ COPY config/settings-reducer.yaml.template /app/config/settings-reducer.yaml
 COPY $REQUIREMENTS /app/config/requirements.txt
 
 # Install developer tools (needed for psutil)
-RUN apt-get update && apt-get install -y python3-dev gcc
+RUN apt-get update && apt-get install -y python3-dev gcc wget
```

Comment:
installing binary wget not good, remove executable after it has been used. (we should have a build layer in the image really)